.github/workflows/ci.yml

```diff
@@ -29,6 +29,10 @@ jobs:
 
   tests:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
```

Comment:
Imagine the following scenario:

- `test:cov` is successful but executes in 10 seconds
- `test:e2e` is NOT successful but executes in 5 seconds

Given this I think the outcome for both would be a failed execution right? So in the end those are not entirely split.

I think it's important that we get independent feedback from both tasks and allow each task to be re-executed in case of failure.

Reply:
In that scenario, acording to the docs (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast), `test:e2e` would fail, and `test:cov` would be cancelled by the CI. We would still having independent feedback for both jobs. In fact, by reading the docs I realised this is the default behavior. Maybe we could set this explicitly to `false` in order to be able to re-run just one job. What do you think?

Reply:
I removed the fail-fast configuration option in [ac37ec1](https://github.com/5afe/safe-client-gateway-nest/pull/173/commits/ac37ec1366ece0316d011ad1e4c0ed4d860e705f). As I said, I'm ok with having an explicit `fail-fast: false` if you find it useful, just let me know 🙂

Reply:
> In that scenario, acording to the docs (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast), `test:e2e` would fail, and `test:cov` would be cancelled by the CI. We would still having independent feedback for both jobs.

But then it's not really independent as one would cancel the other right? 🤔

Reply:
That's right, the first that fails cancels all the other running test executions. I think it's valuable to have them totally independent from each other as you pointed out, changed in [a47ebca](https://github.com/5afe/safe-client-gateway-nest/pull/173/commits/a47ebca22cf7867f0712207413aa025b7a0956e7) 🙂